db/migrate/20210409033915_add_created_in_katello_to_taxonomy.rb

```diff
@@ -0,0 +1,9 @@
+class AddCreatedInKatelloToTaxonomy < ActiveRecord::Migration[6.0]
+  def up
+    add_column :taxonomies, :created_in_katello, :bool, default: false, null: true
```

Comment:
is `null: true` here to make the migration easier since in PostgreSQL adding a null column is cheap? From a data perspective I think NULL and false are treated equal but there is a risk that someone might query for where created_in_katello = false and not get null entries back so there is a risk for subtle bugs, right?

Reply:
Actually I wrote the migration while half asleep so I didn't give it ANY consideration. Good idea to make this non-nullable for the reason you mentioned.